requirements/edx/github.in

```diff
@@ -50,24 +64,13 @@
 #     repos before picking a version number. Don't use 0.0 on master, only for
 #     tight-loop work in progress.
 
-
-# Python libraries to install directly from github
-
-# Third-party:
--e git+https://github.com/edx/django-wiki.git@1.1.1#egg=django-wiki
--e git+https://github.com/openedx/olxcleaner.git@2f0d6c7f126cbd69c9724b7b57a0b2565330a297#egg=olxcleaner
-git+https://github.com/edx/MongoDBProxy.git@d92bafe9888d2940f647a7b2b2383b29c752f35a#egg=MongoDBProxy==0.1.0+edx.2
-
-# original repo is not maintained any more.
-git+https://github.com/edx/django-require.git@0c54adb167142383b26ea6b3edecc3211822a776#egg=django-require==1.0.12
-
-# Our libraries:
--e git+https://github.com/openedx/blockstore.git@1.2.1#egg=blockstore==1.2.1  # Note: Blockstore 1.2.2 is failing.
--e git+https://github.com/edx/codejail.git@3.1.3#egg=codejail==3.1.3
--e git+https://github.com/edx/RateXBlock.git@2.0.1#egg=rate-xblock
--e git+https://github.com/edx-solutions/xblock-google-drive.git@2d176468e33c0713c911b563f8f65f7cf232f5b6#egg=xblock-google-drive
-
-# Third Party XBlocks
-
 git+https://github.com/open-craft/xblock-poll@v1.12.0#egg=xblock-poll==1.12.0
-git+https://github.com/edx-solutions/xblock-drag-and-drop-v2@v2.3.5#egg=xblock-drag-and-drop-v2==2.3.5
+git+https://github.com/openedx/blockstore.git@1.2.1#egg=blockstore==1.2.1  # Note: Blockstore 1.2.2 is failing.
+git+https://github.com/openedx/codejail.git@3.1.3#egg=codejail==3.1.3
+git+https://github.com/openedx/django-require.git@f4f01e4e959adc6210873ae99e7f2c3741afbf35#egg=django-require==1.0.12
```

Comment:
django-require is the only hash that changed -- I updated it to use the latest hash. The rest of the hashes/tags are unchanged.